tests/test_s3_specific.py

```diff
@@ -29,7 +31,9 @@ def test_transfer_config(s3_rig, tmp_path):
 
     # download
     client.set_as_default_client()
-    p = s3_rig.create_cloud_path("dir_0/file0_0.txt")
+    p = s3_rig.create_cloud_path(f"{s3_rig.test_dir}/dir_0/file0_0.txt")
```

Comment:
I don't think you want to add these here. `s3_rig.create_cloud_path` should already include the `test_dir`.

https://github.com/drivendataorg/cloudpathlib/blob/ea3053f24f6425b12662df5489ee5f1f57452875/tests/conftest.py#L77-L82